iidm/iidm-test/src/main/java/com/powsybl/iidm/network/test/FourSubstationsNodeBreakerFactory.java

```diff
@@ -0,0 +1,623 @@
+/**
+ * Copyright (c) 2020, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.test;
+
+import com.powsybl.iidm.network.*;
+import org.joda.time.DateTime;
+
+import java.util.Objects;
+
+/**
+ * This test network is constituted of four substations, with five voltage levels.
+ * Below is a diagram of the network:
+ * <div>
+ *    <object data="doc-files/fourSubstationsNetwork.svg" type="image/svg+xml"></object>
+ * </div>
+ *
+ *
+ * @author Agnes Leroy <agnes.leroy at rte-france.com>
+ */
+
+public final class FourSubstationsNodeBreakerFactory {
+
+    private FourSubstationsNodeBreakerFactory() {
+    }
+
+    public static Network create() {
+        return create(NetworkFactory.findDefault());
+    }
+
+    public static Network create(NetworkFactory networkFactory) {
+        Objects.requireNonNull(networkFactory);
+
+        Network network = networkFactory.createNetwork("fourSubstations", "test");
+        network.setCaseDate(DateTime.parse("2017-06-25T17:43:00.000+01:00"));
+        network.setForecastDistance(0);
+
+        Substation s1 = network.newSubstation()
+                .setId("S1")
+                .setCountry(Country.FR)
+                .add();
+        VoltageLevel s1vl1 = s1.newVoltageLevel()
+                .setId("S1VL1")
+                .setNominalV(225.0)
+                .setLowVoltageLimit(0.0)
+                .setTopologyKind(TopologyKind.NODE_BREAKER)
+                .add();
+
+        BusbarSection busbarSectionFR1 = s1vl1.getNodeBreakerView().newBusbarSection()
+                .setId("S1VL1_BBS")
+                .setName("S1VL1_BBS")
+                .setNode(0)
+                .add();
+        createSwitch(s1vl1, "S1VL1_DI1", "S1VL1_DI1", SwitchKind.DISCONNECTOR, false, false, true, 0, 1);
+        createSwitch(s1vl1, "S1VL1_DI2", "S1VL1_DI2", SwitchKind.DISCONNECTOR, false, false, true, 0, 3);
+        createSwitch(s1vl1, "S1VL1_DI3", "S1VL1_DI3", SwitchKind.BREAKER, false, false, true, 1, 2);
+        createSwitch(s1vl1, "S1VL1_DI4", "S1VL1_DI4", SwitchKind.BREAKER, false, false, true, 3, 4);
+
+        Load load1 = s1vl1.newLoad()
+                .setId("LD1")
+                .setLoadType(LoadType.UNDEFINED)
+                .setP0(25)
+                .setQ0(10.168945)
+                .setNode(4)
+                .add();
+        load1.getTerminal()
+                .setP(25)
+                .setQ(10.168945);
+
+        VoltageLevel s1vl2 = s1.newVoltageLevel()
+                .setId("S1VL2")
+                .setNominalV(225.0)
+                .setLowVoltageLimit(220.0)
+                .setHighVoltageLimit(245.00002)
```

Comment:
Switch voltage limits to have 80% and 120% of the nominal voltage.

It seems your transformer TWT is connected between two voltage levels with the same nominal voltage. Maybe you should change the nominal voltage of all the VL to 400kV except the S1VL1